### PR TITLE
Camel case, you can click on different documents while viewing

### PR DESCRIPTION
**Pull request by kmcostel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -278,7 +282,9 @@
 
       <input class="muliFont" type="button" id="change-prop" value="Submit">
       <br><br>
-      <select id="select-prop" class="muliFont"><option selected>Choose a property</option></select>
```
Comment (lrhruska) [inline index.html:281]: why change the font back to default?

File: public/input_text_load_data.js [hunk 2]
```diff
@@ -111,12 +111,10 @@ function loadData(collection) { //Called from top-level code
...
   }
-  console.log(collection);
 
   $.ajax({
     url: '/v1/search?pageLength=1000',
     data: {
-     // format: 'json',
       collection: collection
     },
     type: 'POST',
```
Comment (lrhruska) [inline public/input_text_load_data.js:120]: Awesome. Good work Kevin. I would merge it but my guess is that @dmcassel will now that he is back

File: index.html (hunk 1 above)
Comment (kmcostel) [inline index.html:281]: Not sure, there seems to be no difference between the code though. I think Ashley might have modified this.
Check out the css for this element on line 245.

File: public/bitemp_graph_helper.js [hunk 3]
```diff
@@ -451,16 +445,16 @@ function setupDelete(chart) {
...
-  else {
+  /*else {
     var lastPeriodLoc = uri.lastIndexOf('.');
     var firstPeriodLoc = uri.indexOf('.');
     if (lastPeriodLoc !== firstPeriodLoc) { //More than one '.', indicates a big number within uri.
       uri = uri.substring(0, firstPeriodLoc) + uri.substring(lastPeriodLoc, uri.length); // Remove the big number.
     }
   }
-  chart.setLogicalURI(uri);
+  chart.setLogicalURI(uri);*/
```
Comment (dmcassel) [inline public/bitemp_graph_helper.js]: Looks like you've commented this block out. Delete?

Comment (kmcostel) [inline public/bitemp_graph_helper.js]: Deleted

**Pull request merged**